src/main/java/net/earthcomputer/clientcommands/command/CGiveCommand.java

```diff
@@ -0,0 +1,52 @@
+package net.earthcomputer.clientcommands.command;
+
+import static com.mojang.brigadier.arguments.IntegerArgumentType.*;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.*;
+import static net.minecraft.command.argument.ItemStackArgumentType.*;
+import static net.minecraft.server.command.CommandManager.*;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.exceptions.*;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.command.argument.ItemStackArgument;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.text.TranslatableText;
+
+public class CGiveCommand {
+
+    private static final SimpleCommandExceptionType NOT_CREATIVE_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.cgive.notCreative"));
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("cgive");
+
+        dispatcher.register(literal("cgive")
+            .then(argument("item", itemStack())
+            .executes(ctx -> give(ctx.getSource(), getItemStackArgument(ctx, "item"), 1))
+                .then(argument("count", integer(1))
+                .executes(ctx -> {
+                    return give(ctx.getSource(), getItemStackArgument(ctx, "item"), getInteger(ctx, "count"));
```

Comment:
Could you make this an expression lambda?

Reply:
Fixed in ebfb51c976bc875e30699603732a1660b12d8d94